app/src/main/java/com/a1ishka/bankaccount/screens/accountDashboard/AccountDashboardScreen.kt

```diff
@@ -67,8 +70,8 @@ fun AccountDashboard() {
                 .padding(top = 50.dp)
         ) {
             Column {
-                AccountCard(account = accountData[2])
-                RecentTransactionTitle()
+                AccountCard(account = accountData[2], onNavigate = { navController.navigate(Screen.AccountAddingScreen.route) })
+                RecentTransactionTitle(onNavigate = { navController.navigate(Screen.AllTransactionScreen.route) })
```

Comment:
it's easier to read the code when set parameters on separate lines (just recommendation)

Reply:
yes, earlier there were simple args, now moved into separates

```            
                AccountCard(
                    account = accountData[2],
                    onNavigate = {
                        navController.navigate(Screen.AccountAddingScreen.route)
                    }
                )
                RecentTransactionTitle(
                    onNavigate = {
                        navController.navigate(Screen.AllTransactionScreen.route)
                    }
                )